.golangci.yml

```diff
@@ -32,10 +32,6 @@ linters:
     - wsl               # too annoying
     - gomodguard        # we are not using modules
 
-run:
-  skip-dirs:
-    - agents/mongodb/internal/
```

Comment:
We still have that directory, and the code there still will not pass linters.

Reply:
Yeah, but it's not synced with upstream code, so we should check it each time we update code.

Reply:
I think you confuse it with Prometheus code what we had in pmm-managed.
This is our own code.

Reply:
So, what's the reason to ignore it?

Reply:
It was copied from qan-agent as is. It can't be improved, it should be rewritten. And running golangci-lint on that directory produces so many warnings that it hides all other problems.

Reply:
We run golangci-lint only for diff between current branch and master, so mostly it won't be visible.

Reply:
As I said before – please run linter locally. This file contains configuration for local linter too.

Reply:
https://github.com/percona/pmm-agent/pull/162/files#diff-b67911656ef5d18c4ae36cb6741b7965R160